cpp/test/assembly/CMakeLists.txt

```diff
@@ -35,3 +35,5 @@ IF (WIN32)
 ELSE()
     target_link_libraries(test_assembly typedb-driver-cpp)
 ENDIF()
+
+target_link_libraries(test_assembly "-framework CoreFoundation")
```

Comment:
This should be for Mac only.

Reply:
Fixed in https://github.com/vaticle/typedb-driver/pull/580/commits/363cd36ed0ef83c4aeb814bec69f6828dcab5e22